### ISSUE TITLE
Projects table view should add aria labels to cell values

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1113]
The cell values are read back as is without reference to the columns, the values should likely contain an aria-label that includes the column name for each cell.

**Comment by mosefer** [comment]
this should be address by jaws/nvda as opposed to the screen reader extension.
